.travis.yml

```diff
@@ -25,6 +25,8 @@ matrix:
       env: SYMFONY_VERSION=2.8.*
     - php: 5.6
       env: SYMFONY_VERSION=3.0.*
+    - php: 5.3
+      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
Yeah, as expect to see more issues because the lower version constraints aren't valid (as they were never tested before this PR).